### DISCUSSION TITLE
SSL necessary?

### DISCUSSION THREAD

**Discussion by p33k** [bauerj/paperless_app#110] [Q&A]
I'm using Paperless in my home network and from outside with VPN.
When I use http with the app will this be a security problem when I'm for example in a hotel wlan and start the app before I'm connected to my vpn?
Will the login credentials be exposed?

**Comment by bauerj** [comment]
Yes, definitely. You should use HTTPS to prevent that.

**Reply by p33k** [reply]
How can I enable ssl with paperless-ngx?
